vendor: bump Pebble to ec6c21662e65

nicktrav opened the pull request:
ec6c2166 sstable: add range key support to BlockIntervalCollector
586718d8 cmd/pebble: support testkeys comparer
b58a7bdc docs/rfcs: pebble table format versions
f3d164d8 db: save level iter debug output before closing
7f1a70dc sstable: reorganize sstable buffers and structs

Update the existing `sstable.NewBlockIntervalCollector` call site to add
the additional required parameter for the range-key collector.

Release note: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/75253)
<!-- Reviewable:end -->

nicktrav commented:
Bumping now in case someone else comes along and is hit with the update tax that should be paid by me :)

jbowens reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 4 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/75253#-:-MtthpA54QOtmQSI7yfs:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @sumeerbhola)



<!-- Sent from Reviewable.io -->

nicktrav commented:
TFTR!

bors r=jbowens

nicktrav committed:
vendor: bump Pebble to ec6c21662e65

ec6c2166 sstable: add range key support to BlockIntervalCollector
586718d8 cmd/pebble: support testkeys comparer
b58a7bdc docs/rfcs: pebble table format versions
f3d164d8 db: save level iter debug output before closing
7f1a70dc sstable: reorganize sstable buffers and structs

Update the existing `sstable.NewBlockIntervalCollector` call site to add
the additional required parameter for the range-key collector.

Release note: none

nicktrav commented:
bors retry

nicktrav commented:
bors cancel

nicktrav commented:
bors r=jbowens
